packages/pluggableWidgets/maps-native/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+## [5.0.0] - 2024-9-05
+
```

Comment:
You don't need to add this line; the automation will update the CHANGELOG with the current version